release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2550,6 +2602,78 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042601[*BZ#2042601*])
+
+* If a `ClusterGroupUpgrade` CR validation fails for any reason, for example, because of an invalid spoke name, no status is available for the `ClusterGroupUpgrade` CR, as if the CR is disabled.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040828[*BZ#2040828*])
+
+* Currently, for `ClusterGroupUpgrade` CR state changes, there is only a single condition type available - `Ready`. The `Ready` condition can have a status of `True` or `False` only. This does not reflect the range of states the `ClusterGroupUpgrade` CR can be in.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042596[*BZ#2042596*])
+
+* The alpha channel for the PTP Operator reports an error that prevents a `PtpConfig` CR from being applied when the `summary_interval` value does not match the value set for `logSyncInterval`. To work around this issue, update all reference PTP configurations to set `summary_interval` and `logSyncInterval` to -4.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040799[*BZ#2040799*])
+
+* When deploying a multi-node cluster on bare-metal nodes, the machine config pool (MCP) adds an additional CRI-O drop-in that circumvents the link:https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/README.md#container-mount-namespace[container mount namespace drop-in]. This results in CRI-O being in the base namespace while the kubelet is in the hidden namespace. All containers fail to get any kubelet-mounted filesystems, such as secrets and tokens.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2028590[*BZ#2028590*])
+
+* Installing {rh-rhacm} 2.5.0 in the `ocm` namespace causes the `infrastructure-operator` pod to fail due to insufficient privileges.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046554[*BZ#2046554*])
+
+* {product-title} limits object names to 63 characters. If a policy name defined in a `PolicyGenTemplate` CR approaches this limit, the Topology Aware Lifecycle Operator cannot create child policies. When this occurs, the parent policy will remain in a `NonCompliant` state.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2057209[*BZ#2057209*])
+
+* In the default ZTP Argo CD configuration, cluster names cannot begin with `ztp`. Using names starting with `ztp` for clusters deployed with Zero Touch Provisioning (ZTP) results in provisioning not completing. As a workaround, ensure that either cluster names do not start with `ztp`, or adjust the Argo CD policy application namespace to a pattern that excludes the names of your clusters but still matches your policy namespace. For example, if your cluster names start with `ztp`, change the pattern in the Argo CD policy app configuration to something different, like `ztp-`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2049154[*BZ#2049154*])
```

Comment:
for SME approval see https://github.com/openshift/openshift-docs/pull/41683/files#r846225920